tests/Resources/Page/ODH/JupyterHub/gpu.resource

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation  Set of Keywords for GPU support
+
+
+*** Keywords ***
+Verify Installed CUDA Version
+    [Documentation]  Checks the installed CUDA version
+    [Arguments]  ${expected_version}
+    Run Cell And Check Output    !nvidia-smi | grep "CUDA Version:" | awk '{split($0,a); print a[9]}'    ${expected_version}
+
+Verify CUDA Is Not Installed
+    [Documentation]  Verifies that CUDA is not installed
+    Run Cell And Check Output    !nvidia-smi    /usr/bin/sh: nvidia-smi: command not found
+
+Verify Pytorch Can See GPU
```

Comment:
In order not to get confused with the Pytorch notebook image, maybe this keyword could be renamed to `Verify Pytorch  Library Can See GPU`  or similar. Same for tensorflow